### ISSUE TITLE
Can't use processors

### ISSUE THREAD

**Issue by Shyam-Chen** [shinnn/vscode-stylelint#170]
```js
// .stylelintrc
{
  "extends": "stylelint-config-standard",
  "processors": ["@mapbox/stylelint-processor-arbitrary-tags"],
  "rules": {}
}
```

```bash
$ stylelint 'src/**/*.{css,vue}'

src/app/hello-world/HelloWorld.vue
 1:1  ✖  Unexpected empty source   no-empty-source
```

VS Code does not show error

![screen shot 2018-04-11 at 11 48 14 am](https://user-images.githubusercontent.com/13535256/38595429-58eb4904-3d7e-11e8-8f4e-a8ba1021b94c.png)

**Comment by Shyam-Chen** [comment]
```diff
// .stylelintrc
{
  "extends": "stylelint-config-standard",
- "processors": ["@mapbox/stylelint-processor-arbitrary-tags"],
  "rules": {}
}
```

```bash
$ stylelint 'src/**/*.{css,vue}'
✨  Done in 1.44s.
```

**Comment by shinnn** [comment]
The result that vscode-stylelint produces looks better for me.  Anyway, [the document of `stylelint-processor-arbitrary-tags`](https://github.com/mapbox/stylelint-processor-arbitrary-tags/tree/0.2.0) says:

> **Do not use this processor with the no-empty-source rule.**
